Makefile.Common

```diff
@@ -58,8 +58,10 @@ fmt: common/gofmt common/goimports common/gofumpt
 .PHONY: modernize
 modernize:
 	$(GOCMD) run \
-		golang.org/x/tools/gopls/internal/analysis/modernize/cmd/modernize@latest \
-		-fix -test -v ./...
+		golang.org/x/tools/go/analysis/passes/modernize/cmd/modernize@latest \
```

Comment:
Rather than using `go run` here, how about we add the package as a tools dependency and use it like the several other tools we already use?
That would allow us to lock into a specific version rather than using latest.